### ISSUE TITLE
2018-Q3 1.1 Compatibility testing survey of top Enterprise sites

### ISSUE THREAD

**Issue by miketaylr** [mozilla/webcompat-team-okrs#18]
See https://wiki.mozilla.org/Compatibility/Enterprise_Testing_Surveys. We'll test the top Enterprise sites provided by Product, and report and categorize Desktop bugs.

* [x] create new wiki page
* [ ] test all sites
* [ ] file bugs
* [ ] write summary report

**Comment by adamopenweb** [comment]
Started outreach for accounts.

**Comment by miketaylr** [comment]
Created https://wiki.mozilla.org/Compatibility/Enterprise_Testing_Surveys.

**Comment by miketaylr** [comment]
cc @softvision-oana-arbuzov
cc @softvision-sergiulogigan

**Comment by miketaylr** [comment]
Update: there's enough to start testing now, while contacts are being found.
Testing: Ready to start testing (splitting time with TP diagnosis) this week.

**Comment by miketaylr** [comment]
Update: started testing, have done Word Online, working on Powerpoint.

**Comment by miketaylr** [comment]
Update: finished Excel, started Outlook testing. Some testing on ServiceNow, started work on Google Drive.

**Comment by miketaylr** [comment]
Update: Outlook, OneNote, OneDrive, Gmail. Continuing the work.

**Comment by miketaylr** [comment]
I don't have a great sense of the % done -- the spreadsheet is hard to parse, I'm guessing 20? @softvision-sergiulogigan @softvision-oana-arbuzov, please change to be more accurate.
